src/Transfer/PickedOptions.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+
+import { findOption, getModeByModifierKey } from './common'
+import { multiSelectedPropType } from '../common-prop-types'
+import { spacers } from '../theme.js'
+
+export const PickedOptions = ({
+    children,
+    dataTest,
+    toggleHighlightedPickedOption,
+    emptySelectionPlaceholder,
+    highlightedPickedOptions,
+    deselectSingleOption,
+}) => (
+    <div data-test={`${dataTest}-pickedoptions`}>
+        {!React.Children.count(children) && emptySelectionPlaceholder}
```

Comment:
For something like this (rendering a fallback), I'm more used to seeing an if at the top that returns the empty placeholder early, and then with the rendering of the children after that.

I don't see a technical downside to doing it this way (except for maybe a call to map without children if there are none, which doesn't bother me). And the benefit is that you only have to define the wrapper once.

I do think I'd personally prefer the if block, and repeating the wrapper. I'm more used to seeing that, so it's easier to parse for me. But that might just be me.

Reply:
Yeah I prefer to keep a single container instead of an if-block, but I guess it's personal preference. I prefer to see at one glance what's going to be rendered. It visually combines the why, what and where.

Also when using the wrapper several times, either there's a slight chance for misalignment due to a type or it slightly increases complexity because of keeping stuff DRY so the misalignment can't happen.

Reply:
I don't have a strong preference for either, so going with your preferred approach seems fine to me 👍